apps/asap-server/test/handlers/teams/index-team-users-handler.test.ts

```diff
@@ -79,7 +79,7 @@ describe('Index Users on Team event handler', () => {
       await indexHandler(event);
 
       expect(algoliaSearchClientMock.saveMany).toHaveBeenCalledWith(
-        usersResponse.items,
+        usersResponse.items.map((data) => ({ data, type: 'user' })),
```

Comment:
Is it worth having a helper function that does this for us? It seems to be repeated rather a lot.

```
const toPayload = (type) => (data) => ({ data, type })
```

```suggestion
        usersResponse.items.map(toPayLoad( 'user' )),
        or
        //set up once per file
        const toPayLoadUser = toPayLoad('user'))
        
        userResponse.items.map(toPayLoadUser)
```

Other place it could be used are for example:
```
{
      data: { ...externalauthorResponse, id: 'external-author-1234' },
      type: 'external-author',
    }

```
or [apps/asap-server/src/handlers/lab/index-lab-users-handler.ts](https://github.com/yldio/asap-hub/pull/1380/files#diff-b8ac5d79ee489f1626903690692f4d014f00fe8dab728e3835504a5b3e0a1be0)
```
await algoliaClient.saveMany(
        foundUsers.items.map((data) => ({
          data,
          type: 'user',
        })),
      );

```